### PR TITLE
Fix/Reviewer console - remove hardcoded reviewer and AC names

### PR DESCRIPTION
**Pull request by xkopenreview**
this pr should remove hardcoded reviewer and AC names from reviewer console

### CODE REVIEW

File: components/webfield/ReviewerConsole.js [hunk 1]
```diff
@@ -392,7 +402,7 @@ const ReviewerConsole = ({ appContext }) => {
               }
             })
             result.groups.forEach((anonGroup) => {
-              if (anonGroup.id.includes(`/Area_Chair_`)) {
+              if (anonGroup.id.includes(`/${areaChairName}_`)) {
```
Comment (melisabok) [inline components/webfield/ReviewerConsole.js:405]: in this case you need to remove the las character if the areaChairName ends with `s`. 

Area_Chairs -> Area_Chair_
Program_Committee -> Program_Committee_

Another option is to pass this as a param: https://github.com/openreview/openreview-py/blob/master/openreview/venue/group.py#L202

**Comment by melisabok** [comment]
Can you please parametrize the name of the review invitation in the reviewers console? "Submit Official Review".

AAAI will use two rounds of review and they will probably use other names.

File: components/webfield/ReviewerConsole.js [hunk 2]
```diff
@@ -559,16 +576,16 @@ const ReviewerConsole = ({ appContext }) => {
       />
       <Tabs>
         <TabList>
-          <Tab id="assigned-papers" active>
-            Assigned Papers
+          <Tab id={`assigned-${submissionName}`} active>
```
Comment (zbialecki) [inline components/webfield/ReviewerConsole.js:579]: I don't think that ids or other values not displayed to the users should be parameterized. These could all stay as "papers" and not have any change in functionality.

**Review by zbialecki — APPROVED** [review]

File: components/webfield/NoteReviewStatus.js [hunk 3]
```diff
@@ -22,6 +22,7 @@ export const ReviewerConsoleNoteReviewStatus = ({
   officialReview,
   invitationUrl,
   reviewDisplayFields,
+  officialReviewName = 'Official_Review',
```
Comment (melisabok) [inline components/webfield/NoteReviewStatus.js:25]: can you read from the webfield properties?

**Comment by melisabok** [comment]
@enrubio could you please test with the AAAI test?

**Review by enrubio — APPROVED** [review]
Looks good! This is what I checked:

Active Consoles:
Screenshot 2024-05-07 at 4 02 44 PM

Console:
Screenshot 2024-05-07 at 4 03 20 PM

Paper with review:
Screenshot 2024-05-07 at 4 05 24 PM

**Comment by melisabok** [comment]
@xkopenreview is it possible to apply a pretty id to the official review invitation that we render in the second column? instead of showing "Submit First_Round_Review", display "Submit First Round Review"

**Pull request merged**